config/common/errors.go

```diff
@@ -51,7 +51,8 @@ var (
 	ErrTooFewMirrorDevices     = errors.New("mirroring requires at least two devices")
 
 	// partition
-	ErrWrongPartitionNumber = errors.New("incorrect partition number; a new partition will be created using reserved label")
+	ErrWrongPartitionNumber   = errors.New("incorrect partition number; a new partition will be created using reserved label")
+	ErrInvalidPartitionNumber = errors.New("partition number cannot be zero while wipe_table is set to false")
```

Comment:
From the reading of the issue, we need to word this in a way that "...its clear that the warning can be avoided for the root disk by specifying it as /dev/disk/by-id/coreos-boot-disk."

Reply:
for the error name we could do : "ErrInvalidPartitionConfig" or is that too general?